### PR TITLE
docs: installing podman with podman desktop - references #4013

### LINKED ISSUE #4013: Docs: update Installing on macOS

### LINKED ISSUE #4124: Update Onboarding Docs
Document the new onboarding feature: what users will see, how to work with it, etc

### PR DESCRIPTION
**Pull request by themr0c**
Rework the installing podman with podman desktop page to make it valid for Windows and macOS.

First step to get the "installing podman" section out of macos install.

references #4013

### CODE REVIEW

File: website/docs/onboarding/containers/installing-podman.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Podman Desktop can assist you to install the Podman container engine in a Fedora distribution of Linux, on a virtual machine.
+
+Main benefits are:
+
+- Ease of use.
+- On Windows: Windows Subsystem for Linux version 2 (WSL 2) native virtualization performance.
+
+Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if:
+
+- You prefer working with a RHEL distribution of Linux.
```
Comment (mairin) [inline website/docs/onboarding/containers/installing-podman.md]: I think the podman-machines are Fedora based though, not RHEL. Maybe ¨RHEL family" but I'm not sure if that makes sense either. 🤔

File: website/docs/onboarding/containers/installing-podman.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Your environment doesn't allow you to meet the prerequisites.
+
+#### Prerequisites
+
+- Podman is not installed.
+- 6 GB RAM.
+- The host runs Windows or macOS.
+- Windows prerequisites:
+  - No VPN.
```
Comment (mairin) [inline website/docs/onboarding/containers/installing-podman.md]: Is no VPN still the case post podman 4.6?

File: website/docs/onboarding/containers/installing-podman.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  - [WSL prerequisites](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+    - User with administrator privileges.
+    - Windows 64bit.
+    - Windows 10 Build 18362 or greater, or Windows 11.
+    - On a virtual machine: [Nested Virtualization enabled](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization).
+  - No WSL 2 Linux virtual machine is running.
+
+#### Procedure
+
+1. (On Windows) Enable the WSL feature without installing the default Ubuntu distribution of Linux.
```
Comment (mairin) [inline website/docs/onboarding/containers/installing-podman.md:36]: Sorry if a dupe, it seems GitHub ate this comment earlier - see https://github.com/containers/podman-desktop/pull/4209 - Luca has it so if the onboarding pre-requisite check doesn't see WSL2 it offers to install. It seems worth mentioning? But I am not sure what is the better experience, for users to set it up themselves and we use the pre-requisite check to catch a few who missed that step, or just let all users rely on it?

Comment (themr0c) [reply website/docs/onboarding/containers/installing-podman.md:36]: Yes we discussed that and the decision was:
- Keep now.
- Remove once new onboarding that automates WSL installation is out.

File: website/docs/onboarding/containers/installing-podman.md (hunk 2 above)
Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]: Indeed, that can go away.

```suggestion
```

File: website/docs/onboarding/containers/installing-podman.md (hunk 1 above)
Comment (cfergeau) [reply website/docs/onboarding/containers/installing-podman.md]: The podman bundlles used by CRC are based on Fedora CoreOS.
For podman on RHEL, starting a MicroShift instance but using it with podman-remote should work.

Comment (themr0c) [reply website/docs/onboarding/containers/installing-podman.md]: OK then I rmove this line.

File: website/docs/onboarding/containers/installing-podman.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Main benefits are:
+
+- Ease of use.
+- On Windows: Windows Subsystem for Linux version 2 (WSL 2) native virtualization performance.
+
+Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if:
+
+- You prefer working with a RHEL distribution of Linux.
+- Your environment doesn't allow you to meet the prerequisites.
```
Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]:
```suggestion
Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if your environment doesn't allow you to meet the prerequisites.
```

File: website/docs/onboarding/containers/installing-podman.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+1. (On Windows) Enable the WSL feature without installing the default Ubuntu distribution of Linux.
+   See [Enabling WSL 2](https://docs.microsoft.com/en-us/windows/wsl/install) and [WSL basic commands](https://learn.microsoft.com/en-us/windows/wsl/basic-commands):
+
+   ```powershell
+   wsl --install --no-distribution
+   ```
+
+2. The **Home** screen displays _Podman Desktop was not able to find an installation of Podman_.
+   Click on **Install**.
```
Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]:
```suggestion
   Click **Install**.
```

File: website/docs/onboarding/containers/installing-podman.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+3. Podman Desktop checks the prerequisites to install Podman.
+   When necessary, follow the instructions to install prerequisites.
+
+4. Podman displays the dialog: _Podman is not installed on this system, would you like to install Podman?_.
+   Click on `Yes` to install Podman, and follow the installation program instructions.
```
Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]:
```suggestion
   Click `Yes` to install Podman, and follow the installation program instructions.
```

File: website/docs/onboarding/containers/installing-podman.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+5. Click on **Initialize Podman**.
```
Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]:
```suggestion
5. Click **Initialize Podman**.
```

File: website/docs/onboarding/containers/installing-podman.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+#### Prerequisites
+
+- Podman is not installed.
+- 6 GB RAM.
+- The host runs Windows or macOS.
+- Windows prerequisites:
+  - [WSL prerequisites](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+    - User with administrator privileges.
+    - Windows 64bit.
+    - Windows 10 Build 18362 or greater, or Windows 11.
```
Comment (lstocchi) [inline website/docs/onboarding/containers/installing-podman.md]: The build version must be updated to 19043 -> https://github.com/containers/podman-desktop/pull/4255

Comment (themr0c) [inline website/docs/onboarding/containers/installing-podman.md]:
```suggestion
    - Windows 10 Build 19043 or greater, or Windows 11.
```

Comment (themr0c) [reply website/docs/onboarding/containers/installing-podman.md]: Updated.

**Review by lstocchi — APPROVED** [review]
Maybe the procedure will change with the dashboard refactoring + onboarding workflow but LGTM for now

**Pull request merged**